### ISSUE TITLE
Document default value for expression_kinds of MA0003

### ISSUE THREAD

**Issue by drieseng** [meziantou/Meziantou.Analyzer#352]
**Context**
Rule identifier: _MA0003_

**Describe the "bug" (request)**
I expected **MA0003** to also be reported in case of a numeric literal. After taking a brief look at **NamedParameterAnalyzer**, by default **MA0003** appears to only be reported for **null** and **bool** literals. That is, unless the called method was decorated with **RequireNamedArgumentAttribute**.

**To Reproduce**
Code or steps to reproduce the behavior:

````c#
public void Execute()
{
    Do(5);
}

private void Do(int value)
{
}
````

**Expected result:**

**MA0003** is reported for passed *5* as unnamed argument.

**Actual result:**

**MA0003** is not reported.

**Extra info:**

You built a great collection of analyzers. Thanks!!!

**Comment by meziantou** [comment]
You can configure MA0003 to report numeric values using the editorconfig as explained in the documentation. It is not enabled by default, as it is very noisy.

https://github.com/meziantou/Meziantou.Analyzer/blob/main/docs/Rules/MA0003.md#configuration-editorconfig

````
MA0003.expression_kinds = Null, Boolean, Numeric, String
````

**Comment by drieseng** [comment]
@meziantou, thanks for the quick turnaround!
